.github/CODEOWNERS

```diff
@@ -5,6 +5,8 @@
 
 /content/build/ @crazy-max @dvdksn
 
+/content/build-cloud/ @dvdksn
```

Comment:
@fiam @silvin-lubecki Fine to add you there?

Reply:
added, lmk if you want out :-D

Reply:
Sounds good to me, thank you!